.infrastructure/configMap.yml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: todoapp-congifmap
```

Comment:
There is a typo in the `name` field. It should be `todoapp-configmap` instead of `todoapp-congifmap`.